Makefile

```diff
@@ -13,7 +13,7 @@ up: docker-compose.yaml docker-compose.override.yaml ## Does docker-compose up,
 clean: ## Deletes all containers and volumes. WILL DROP ALL DB DATA
 	docker-compose down --volumes --remove-orphans
 
-stop: ## Stops all containers
+down: ## Stops all containers
```

Comment:
Down and stop have a different meaning in this context. As up and start does.

Up = create and start
Start = start
Down = stop and tear down
Stop = stop

I agree it's a little messy. But IMHO this edit is not making it better.